low-coverage/script.whizzml

```diff
@@ -0,0 +1,20 @@
+(define (low-coverage dsid threshold)
+  (let (ds (fetch dsid)
+        size (ds "rows")
+        name (ds "name")
+        fields (keys (ds "fields"))
+        missing (map (lambda (x) (ds ["fields" 
+                                          x 
+                                          "summary" 
+                                          "missing_count"])) 
+                         fields)
+        missing-pairs (iterate (a [] x fields y missing) (append a [x y]))
+
+        low-list (filter (lambda (x) (< (last x) (* (- 1 threshold) size))) 
+                         missing-pairs)
+        fields-to-keep (map (lambda (x) (head x)) low-list))
```

Comment:
why 4 iterations? it can be all computed in one go, no?:

```
fields (ds "fields")
mc (* (- 1 threshold) (ds "rows"))
to-keep (filter (lambda (id) (< (fields [id "summary" "missing_count"]) mc))
                (keys fields))
```